demo/package.json

```diff
@@ -16,8 +16,10 @@
     "export": "next export"
   },
   "dependencies": {
-    "@moxy/react-lib-template": "file:..",
+    "@moxy/next-scroll-behavior": "file:..",
+    "@moxy/react-page-swapper": "^1.0.0",
     "next": "^9.3.5",
-    "onchange": "^6.1.0"
+    "onchange": "^6.1.0",
```

Comment:
This should be a dev dependency, can you change it here and in the react-lib template?

Reply:
Done ✅